package.json

```diff
@@ -113,19 +113,19 @@
     "postcss-cli": "^9.1.0",
     "postcss-color-function": "^4.1.0",
     "postcss-custom-media": "^8.0.0",
-    "postcss-each": "^0.10.0",
+    "postcss-each": "^1.1.0",
```

Comment:
Just a heads up, this version seems to be missing a lot of functionality that we need
https://github.com/teamleadercrm/ui/pull/1742